test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -231,6 +140,45 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 		})
 	})
 
+	ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTerminal=false", func() {
```

Comment:
Diff -w between test chunks:

```diff
diff --git a/tmp/foo8 b/tmp/foo7
index bf123ba..9d469fb 100644
--- a/tmp/foo8
+++ b/tmp/foo7
@@ -1,11 +1,11 @@
-ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTerminal=true", func() {
-
+       ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTerminal=false", func() {
                const (
                        pollInterval                        = 1 * time.Second
                        nodeStatusUpdateTimeout             = 30 * time.Second
                        nodeShutdownGracePeriod             = 20 * time.Second
                        nodeShutdownGracePeriodCriticalPods = 10 * time.Second
                )
+
                tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
                        initialConfig.FeatureGates = map[string]bool{
                                string(features.GracefulNodeShutdown):                   true,
@@ -13,7 +13,9 @@ ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTermina
                        }
                        initialConfig.ShutdownGracePeriod = metav1.Duration{Duration: nodeShutdownGracePeriod}
                        initialConfig.ShutdownGracePeriodCriticalPods = metav1.Duration{Duration: nodeShutdownGracePeriodCriticalPods}
+                       initialConfig.ShutdownSetPodsPhaseToTerminal = false
                })
+
                ginkgo.BeforeEach(func() {
                        if err := lookEmitSignalCommand(); err != nil {
                                e2eskipper.Skipf("skipping test because: %v", err)
@@ -22,6 +24,7 @@ ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTermina
                        ginkgo.By("Wait for the node to be ready")
                        waitForNodeReady()
                })
+
                ginkgo.AfterEach(func() {
                        ginkgo.By("Emitting Shutdown false signal; cancelling the shutdown")
                        err := emitSignalPrepareForShutdown(false)
@@ -29,5 +32,7 @@ ginkgo.Context("when gracefully shutting down with ShutdownSetPodsPhaseToTermina
                })
 
                ginkgo.It("should be able to gracefully shutdown pods with various grace periods", func() {
-                       runTestShutdownPodsWithVariousGracePeriods(f, true)
+                       runTestShutdownPodsWithVariousGracePeriods(f, false)
+               })
+
        })
```

:+1: